packages/analytics-connector/src/eventBridge.ts

```diff
@@ -9,16 +14,58 @@ export type AnalyticsEventReceiver = (event: AnalyticsEvent) => void;
 export interface EventBridge {
   logEvent(event: AnalyticsEvent): void;
   setEventReceiver(listener: AnalyticsEventReceiver): void;
+  setApiKey(apiKey: string): void;
 }
 
 export class EventBridgeImpl implements EventBridge {
+  private apiKey = '';
   private receiver: AnalyticsEventReceiver;
-  private queue: AnalyticsEvent[] = [];
+  private inMemoryQueue: AnalyticsEvent[] = [];
+  private globalScope = getGlobalScope();
+
+  private getStorageKey(): string {
+    return `EXP_unsent_${this.apiKey.slice(0, 10)}`;
```

Comment:
I dont think we should use the apikey here. Could just use the instance name which is associated with the analytics Connector that initializes this. 

Right now there's some issues with apiKey in this class. What if we intiailize two instances with the same instance name and different api keys?